src/apiCalls/apiCalls.js

```diff
@@ -9,5 +9,16 @@ function getAllMovies() {
   );
 }
 // console.log('fetchedData', data)
+function getIndividualMovie(id) {
```

Comment:
This looks like it might be a second apiCalls.js file - is that intentional or something that got duplicated by mistake?